Desktop/gui/preferencesmodel.cpp.in

```diff
@@ -257,6 +261,13 @@ void PreferencesModel::FUNC_NAME(TYPE newVal)							\
 	emit NOTIFY();														\
 }
 
+void PreferencesModel::setTempDeveloperFolder(QString devFolder)
+{
+	if (_developerFolder == devFolder) return;
```

Comment:
If `_developerFolder` is set through `setTempDeveloperFolder` a better name would be `_tmpDevFolder`